src/app/places/places.component.ts

```diff
@@ -66,46 +68,49 @@ export class PlacesComponent implements OnInit {
   }
 
   ratingFromHigh() {
-    this.PlacesFromService.sort((a, b) =>
+    this.Places.sort((a, b) =>
       a.Rating < b.Rating ? 1 : b.Rating < a.Rating ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'rate-high' } });
   }
 
   ratingFromLow() {
-    this.PlacesFromService.sort((a, b) =>
+    this.Places.sort((a, b) =>
       a.Rating > b.Rating ? 1 : b.Rating > a.Rating ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'rate-low' } });
   }
 
   reset() {
-    this.PlacesFromService.sort((a, b) =>
-      a.Id > b.Id ? 1 : b.Id > a.Id ? -1 : 0
-    );
+    this.Places.sort((a, b) => (a.Id > b.Id ? 1 : b.Id > a.Id ? -1 : 0));
     this.page = 1;
     this.router.navigate(['/'], { queryParams: {} });
   }
 
   asc() {
-    this.PlacesFromService.sort((a, b) =>
+    this.Places.sort((a, b) =>
       a.Name > b.Name ? 1 : b.Name > a.Name ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'alf-asc' } });
   }
 
   desc() {
-    this.PlacesFromService.sort((a, b) =>
+    this.Places.sort((a, b) =>
       a.Name < b.Name ? 1 : b.Name < a.Name ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'alf-dsc' } });
   }
 
-  private fetchPlaces(): any {
-    this.PlacesFromService = this.PlacesService.getPlaces();
+  rangeChange(event: RangeInterface) {
```

Comment:
I would change this method name to `filterPlaces`

Reply:
Changed :-)